lib/dradis/plugins/ntospider/field_processor.rb

```diff
@@ -2,7 +2,11 @@ module Dradis::Plugins::NTOSpider
   class FieldProcessor < Dradis::Plugins::Upload::FieldProcessor
 
     def post_initialize(args={})
-      @nto_object = ::NTOSpider::Vuln.new(data)
+      if data.name == 'Vuln'
```

Comment:
I wanted to see if we could meta-program this. And we can but I don't like it, so it's not a suggestion. Just a note.

```ruby
@nto_object = "::NotSpider::#{data.name}".safe_constantize&.new(data)
```